server/lib/market.coffee

```diff
@@ -22,66 +24,47 @@ comparator = (a, b) ->
     else
         return 0
 
-readApps = (cb) ->
-    fs.readdir './market/apps', (err, files) ->
-        log.error "[Error] Read market: #{err}" if err?
-        return cb(err) if err?
-        apps = []
-        for file in files
-            try
-                # Node js
-                apps.push require "../../../market/apps/#{file}"
-            catch
-                # Coffeescript
-                apps.push require "../../market/apps/#{file}"
-        apps.sort comparator
-        cb null, apps
-
-
 download = module.exports.download = (callback) ->
+
     isDownloading = true
 
     # Retrieve market path
     if process.env.MARKET?
         # Use a specific market
-        url = "https://gitlab.cozycloud.cc/zoe/cozy-registry.git"
-        branch = process.env.MARKET
+        path = process.env.MARKET
     else
         # Use default market
-        url = "https://github.com/cozy/cozy-registry.git"
-        branch = "master"
-
-    # Clone market (cannot use github API due to rate limit)
-    command =  "git clone #{url} markettmp && " + \
-          "cd markettmp && " + \
-          "git checkout #{branch}"
-    # Clone new market in temporary folder
-    exec command, {}, (err, stdout, stderr) ->
-
-        if err?
-            # If clone doesn't work, keep old market
-            log.error "[Error] Clone market: #{err}" if err?
-            del './markettmp', (error) ->
-                callback(err) if err?
+        path = "https://registry.cozycloud.cc/cozy-registry.json"
```

Comment:
@poupotte maybe we should call that `marketUrl`. `path` makes me thing to the path module of nodejs, and to be pedantic, it's more an URL than a path.